proto/rill/runtime/v1/queries.proto

```diff
@@ -54,6 +54,22 @@ service QueryService {
     };
   }
 
+  // MetricsViewComparison returns a toplist containing comparison data of another toplist (same dimension/measure but a different time range).
+  // Returns a toplist without comparison data if comparison time range is omitted.
+  //
+  // ie. comparsion toplist:
+  // | measure1_base | measure1_previous   | measure1__delta_abs | measure__delta_rel | dimension |
+  // |---------------|---------------------|---------------------|--------------------|-----------|
+  // | 2             | 2                   | 0                   | 0                  | Safari    |
+  // | 1             | 0                   | 1                   | N/A                | Chrome    |
+  // | 0             | 4                   | -4                  | -1.0               | Firefox   |
+  //
+  // ie. toplist:
+  // | measure1 | measure2 | dimension |
+  // |----------|----------|-----------|
+  // | 2        | 45       | Safari    |
+  // | 1        | 350      | Chrome    |
+  // | 0        | 25       | Firefox   |
```

Comment:
Omg amazing! this goes so far above and beyond what I expected, and is so informative and helpful!!! <3  <3  <3